.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,55 @@
+name: Benchmark
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
```

Comment:
why not on every branch? there's no allotment on actions usage.

Reply:
only interesting in tracking things against master - as both the comparison point and points on a graph.

PRs will still measure but not log the data